sermant-backend/src/main/webapp/frontend/src/views/ConfigInfo.vue

```diff
@@ -161,20 +161,38 @@
                        && requestParam.pluginType != 'tag-transmission'">
         <el-descriptions title="服务信息" style="margin-top: 15px;">
         </el-descriptions>
-        <el-form-item prop="serviceName">
+        <el-form-item prop="serviceName" v-if="requestParam.pluginType == 'flowcontrol'">
+          <el-input size="large" v-model.trim="requestParam.serviceName" placeholder="请输入服务名称"
+                    style="width: 90%;margin-right: 5px;" :disabled="types.modifyFlay" @change="handlerChange(false)">
+            <template #prepend>
+              <span color="#606266" style="width: 100px;text-align: center;">service</span>
+            </template>
+          </el-input>
+          <el-tooltip class="box-item" effect="dark"
+                      content="微服务的名称，由微服务配置文件的dubbo.application.name、spring.applicaton.name确定"
+                      placement="right-start">
+            <el-icon color="lightgray" size="20">
+              <Warning/>
```

Comment:
info

Reply:
Waring is an icon, there is no info icon.

Reply:
use question `?`